cypress/e2e/google.cy.ts

```diff
@@ -1,79 +1,38 @@
-describe("Log in feature", () => {
-  beforeEach(() => {
-    cy.visit("https://www.demoblaze.com/index.html");
-    
-    cy.get("#login2").click();
-    
-    cy.get("#loginusername").type("aleechetorres",{force:true});
-    cy.get("#loginpassword").type("1234567",{force:true});
-    cy.get(".btn-primary").contains("Log in").click();
+import {LoginPage, CategoriesPage, CartPage, PurchasePage, Contact} from '../pages/index'
 
-    // Validar que el login y usuario username corresponden a la sesión iniciada
-    cy.get("#nameofuser").should('contain.text', "Welcome aleechetorres");
-  });
+const loginPage = new LoginPage();
+const categoriesPage = new CategoriesPage();
+const cartPage = new CartPage();
+const purchasePage = new PurchasePage();
+const contact = new Contact();
 
-  it("Añadir celular al carrito y validar que el nombre corresponde al item seleccionado", () => {
-    cy.get(".card-title").contains("Iphone 6 32gb").click();
-    cy.get("#tbodyid").should("contain", "Iphone 6 32gb");
-    cy.get(".btn-lg").contains("Add to cart").click();
-    
-  });
+const products = Cypress.env("Products");
+
+const phoneElement = products.phoneElement;
+const laptopElement = products.laptopElement;
+const monitorElement = products.monitorElement;
 
-  it("Añadir Laptop al carrito y validar que el nombre corresponde al item seleccionado", () => {
-    cy.get(".nav-link").contains("Home").click();
-    cy.get(".list-group-item").contains("Laptops").click();
-    cy.get(".card-title").contains("MacBook Pro").click();
-    cy.get("#tbodyid").should("contain", "MacBook Pro");
-    cy.get(".btn-lg").contains("Add to cart").click();
-    
+const elementToAdd = [phoneElement, laptopElement, monitorElement];
+
+describe("Log in feature", () => {
+  beforeEach(() => {
+    cy.visit("https://www.demoblaze.com/index.html");
+    loginPage.verfifylogInOption();
   });
 
-  it("Añadir Monitor al carrito y validar que el nombre corresponde al item seleccionado", () => {
-    cy.get(".nav-link").contains("Home").click();
-    cy.get(".list-group-item").contains("Monitors").click();
-    cy.get(".card-title").contains("Apple monitor 24").click();
-    cy.get("#tbodyid").should("contain", "Apple monitor 24");
-    cy.get(".btn-lg").contains("Add to cart").click();
-    
+  it("Añadir elementos al carrito y validar que el nombre corresponde al item seleccionado", () => {
+    categoriesPage.addElementsToCart(elementToAdd);
```

Comment:
divide esta función para que en el test se vean los pasos... algo como:

`visitHomePage();`
`addElementToCart()`
`verifyElementTitle()`

Es solo un ejemplo, pero no es común encontrar un test con una sola línea de código jajaja además que queda mucho más descriptivo el paso a paso de lo que estás haciendo

Reply:
Se divide la función en: 

- Add products to cart
- Verify that Product Tittle corresponds